build_debian.sh

```diff
@@ -240,6 +240,9 @@ sudo LANG=C chroot $FILESYSTEM_ROOT easy_install pip
 sudo LANG=C chroot $FILESYSTEM_ROOT pip install 'docker-py==1.6.0'
 ## Note: keep pip installed for maintainance purpose
```

Comment:
since you upgrade the access to the redis DB, may I suggest to also add here python redis module installation?
`sudo LANG=C chroot $FILESYSTEM_ROOT pip install redis`